misc/Makefile.am

```diff
@@ -35,3 +35,4 @@ install-data-hook:
 	-systemctl daemon-reload
 
 endif
+bin_SCRIPTS = cf-support
```

Comment:
`bin_SCRIPTS` seems to work on ubuntu in a local VM but in CI seems to fail, maybe use `dist_bin_SCRIPTS` as that option doesn't expect to "build" `cf-support`? Will try tomorrow.